### ISSUE TITLE
Missing package dependency

### ISSUE THREAD

**Issue by rafamanzo** [analizo/analizo#115]
Running version 1.20.7 on Debian Stretch causes the following error

```
root@5b5ac34dac24:/# analizo metrics
Can't locate Env/Path.pm in @INC (you may need to install the Env::Path module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/
local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_
perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Analizo/Command.pm line 6.
BEGIN failed--compilation aborted at /usr/share/perl5/Analizo/Command.pm line 6.
Compilation failed in require at /usr/share/perl5/Analizo.pm line 2.
BEGIN failed--compilation aborted at /usr/share/perl5/Analizo.pm line 2.
Compilation failed in require at /usr/bin/analizo line 3.
BEGIN failed--compilation aborted at /usr/bin/analizo line 3.
```

Installing `libenv-path-perl` fixes it.

**Comment by eserte** [comment]
It seems that File::Slurp is also missing.

**Comment by joenio** [comment]
yes, File::Slurp was solved on issue #114
